README.md

```diff
@@ -11,7 +11,7 @@ Summary: A Python module for hyperspectral image processing.
 
 A pure Python module for processing hyperspectral image data (imaging spectroscopy data).
 It has functions for reading, displaying, manipulating, and classifying hyperspectral
-imagery. SPy is Free, Open Source Software (FOSS) distributed under the MIT License..
+imagery. SPy is Free, Open Source Software (FOSS) distributed under the GNU General Public License.
```

Comment:
@tboggs as you can see.... again the bot has picked up the incorrect license from somewhere.

Reply:
@conda-forge-admin can you please take a look and advise on why this is happening/ Thank you

Reply:
@lewismc Yes, I see it. This page indicates that it is waiting for my review to accept the PR. Should I merge it (again) or wait for @conda-forge-admin to chime in here?

Reply:
Let's wait for @conda-forge-admin to chime in here?

Reply:
@conda-forge-admin can you please provide some guidance here? We're at a loss as to what keeps overwriting the README.